code/modules/vehicles/wheelchair.dm

```diff
@@ -39,9 +43,11 @@
 	AddComponent(/datum/component/simple_rotation) //Since it's technically a chair I want it to have chair properties
 	AddElement(/datum/element/noisy_movement, volume = 75)
 
-/obj/vehicle/ridden/wheelchair/atom_destruction(damage_flag)
-	new /obj/item/stack/rods(drop_location(), 1)
-	new /obj/item/stack/sheet/iron(drop_location(), 1)
+/obj/vehicle/ridden/wheelchair/handle_deconstruct(disassembled)
+	remove_bell()
+	remove_bomb()
+	new /obj/item/stack/rods(drop_location(), disassembled ? 6 : 1)
+	new /obj/item/stack/sheet/iron(drop_location(), disassembled ? 4 : 1)
```

Comment:
this seems prone to potential duplication exploits if we ever change the construction costs of making wheelchairs
Is there a better way to do this?

Reply:
Not that I can think of easily to be honest, objects don't know what materials they are crafted from
I also didn't add this, I only refactored it to use better procs